packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -157,16 +197,24 @@ def setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net):
             if same_host:
                 proxy_host = origin_host
             cmd = '/opt/mesosphere/bin/curl -s -f -m 5 http://{}/test_uuid'.format(vipaddr)
-            origin_app, origin_app_uuid = vip_app(container, vip_net, origin_host, vip)
-            proxy_app, proxy_app_uuid = vip_app(container, proxy_net, proxy_host, None)
-            # allow these apps to run on public slaves
-            origin_app['acceptedResourceRoles'] = ['*', 'slave_public']
-            proxy_app['acceptedResourceRoles'] = ['*', 'slave_public']
-            # We do not need the service endpoints because we have deterministically assigned them
-            for app_definition in (origin_app, proxy_app):
-                r = dcos_api_session.marathon.post('v2/apps', json=app_definition)
-                r.raise_for_status()
-                apps.append(app_definition)
+            if pod:
```

Comment:
See above. Would prefer a whole new method than a bunch of if statements when pods don't even support the container parameter